trackr-app/app/src/main/java/com/trackr/trackr_app/ui/add/AddScreen.kt

```diff
@@ -6,47 +6,48 @@ import androidx.compose.material.*
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.Check
 import androidx.compose.runtime.*
-import androidx.compose.runtime.livedata.observeAsState
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.navigation.NavHostController
 import com.trackr.trackr_app.ui.shared.InputWidget
 import com.trackr.trackr_app.ui.shared.InteractiveDropdownWidget
+import com.trackr.trackr_app.ui.theme.Rubik
 import com.trackr.trackr_app.viewmodels.AddScreenViewModel
+import java.time.format.TextStyle
+import java.util.*
+
+
+val MONTHS = listOf(
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec"
+)
+
+val REMINDER_INTERVALS = listOf(
+    "1 day before", "3 days before",
+    "1 week before", "2 weeks before", "1 month before"
+)
```

Comment:
Would I make more sense to throw these into the viewmodel instead (and have some sort of get valid months function), since it's the viewmodel that will actually be using these values?

I understand that it works either way, as the added event values eventually get passed through, I'm just wondering what's cleaner/more logical.

Also, if we were to throw these in the viewmodel, then we could control what months are available for the user to select (no months before the current month, for example).

Reply:
yeah agreed, i'll change that